src/main/kotlin/com/manimdsl/executor/DataStructureShapes.kt

```diff
@@ -68,6 +69,33 @@ sealed class BoundaryShape(var x1: Double = 0.0, var y1: Double = 0.0) {
         return this
     }
 }
+object NoBoundary: BoundaryShape() {
+    override var width: Int
```

Comment:
I'm guessing there isn't a shorter way of doing this that doesn't break the current structure of `BoundaryShape`?